includes/html/pages/device/showconfig.inc.php

```diff
@@ -20,9 +20,21 @@
             echo generate_link('Latest', ['page' => 'device', 'device' => $device['device_id'], 'tab' => 'showconfig']);
         }
 
-        if (Config::get('rancid_repo_type') == 'svn' && function_exists('svn_log')) {
+        if (Config::get('rancid_repo_type') == 'svn') {
             $sep = ' | ';
-            $svnlogs = svn_log($rancid_file, SVN_REVISION_HEAD, null, 8);
+
+            $process = new Process(['svn', 'log', '-l 8', '-q', '--xml', $rancid_file], $rancid_path);
```

Comment:
There is a good change svn may not be in path.  Best to have the full path here... LibreNMS has some executable finder code.

Reply:
Thank you for the recommendation. I have updated to using Config::locateBinary('svn') to get the full path of 'svn'